llm_benchmark/benchmark/vllm_benchmark/benchmark_serving.py

```diff
@@ -880,9 +1009,14 @@ def __init__(self, model, input_len, output_len, num_prompts, base_url):
             self.dataset = None
             self.dataset_name = "random"
             self.dataset_path = None
-            self.sharegpt_output_len = None
-            self.sonnet_input_len = 550
-            self.sonnet_output_len = 150
+            self.input_column = "input"
+            self.output_column = None
+            self.is_hf = False
+            self.mean_input_len = 200
+            self.std_input_len = 50
+            self.mean_output_len = 100
+            # self.sonnet_input_len = 550
+            # self.sonnet_output_len = 150
             self.sonnet_prefix_len = 200
```

Comment:
does sonnet prefix len has any other purpose when compared to the input len? If not then we could use the same for sonnet as well

Reply:
The prefix length is not mandatory. If we choose to remove it, we can directly use the llmperf sonnet implementation.